.github/workflows/build-verification.yml

```diff
@@ -60,6 +66,8 @@ jobs:
       fail-fast: false
       matrix:
         versions: ${{ fromJson(needs.generate_versions.outputs.matrix) }}
+        java: [ { jdk: 11, jdkImageIssue: 11.0.12 ,jdkImageIssueAlt:  11.0.13 }, { jdk: 17,  jdkImageIssue:  17.0.3+7 ,jdkImageIssueAlt: 17.0.5+8},  { jdk: 19,  jdkImageIssue:  19.0.3+7 ,jdkImageIssueAlt: 19.0.5+8} ]
```

Comment:
I think I failed to see the bigger picture here earlier.
The JDKImage tests are part of the `testAndroidXXX` matrix. This means that the entire matrix is launched with the AGP versions and the differing JDKImage versions. In my previous comment I thought it would be redundant to test the different minor versions of the JDK with each JDKImage issue but since they are all part of the same matrix it would be hard to untangle.

I propose just testing JDK 11 and JDK 19 and removing JDK 17. Sorry to make you undo this: for both JDK 11 and JDK 19 let's also test the different minor versions of the JDKImage issue.